### ISSUE TITLE
trainning loss can't convergence

### ISSUE THREAD

**Issue by hongyan19890126** [GXYM/DRRG#9]
I have a problem ,i using icdar2015 for trainning,but i found after 400 steps,model still can not convergence, trainning loss is still almost 1.5,so could you get me some advise。my config is  optimizer = dict(type='SGD',lr = 0.01, momentum=0.9, weight_decay=1e-4) my 
lr_config is step=[50, 100,200,300, 600].

**Comment by GXYM** [comment]
> weight_decay

If you use the pre-training model, you can try to modify the learning rate. If not, the Adam optimizer is recommended. In addition, you can try to modify “weight_decay” to a smaller value
